### DISCUSSION TITLE
小白一个，这个项目如何编译跑起来，有没有相关的教程

### DISCUSSION THREAD

**Discussion by MichaelYiyouson** [chenshuo/muduo#598] [General]
GitHub issues are for tracking bugs, not for general discussing like a forum.

If you have a general question to ask, send it to mailing list:

muduo-library@googlegroups.com

Or visit:
https://groups.google.com/forum/#!forum/muduo-library

When file an issue muduo, please provide a [SSCCE](http://sscce.org):
Short, Self Contained, Correct (Compilable), Example.

Also specify the exact environment where the issue occurs:

**Comment by mamil** [comment]
ref to https://github.com/chenshuo/muduo-tutorial
